js/settings.js

```diff
@@ -0,0 +1,21 @@
+$(document).ready(function(){
+        var smsSettings = {
+                save : function() {
+                        var data = {
+                                savePath : $('#phone-number').val()
+                        };
+                        OC.msg.startSaving('#twofactor-sms-personal .msg');
+                        $.post(OC.filePath('twofactor_sms', 'ajax', 'settings/personal.php'), data, smsSettings.afterSave);
```

Comment:
any specific reason why you chose a ``/ajax/`` url? I'd prefer to remove that prefix